cluster/senza-definition.yaml

```diff
@@ -42,7 +42,7 @@ SenzaComponents:
           LoadBalancerPort: 443
       Tags:
         - Key: "KubernetesCluster"
-          Value: {Ref: 'AWS::StackName'}
+          Value: kubernetes
```

Comment:
we can run multiple versions in one account/subnet (e.g. for testing or migrations)